src/basic/env-file.c

```diff
@@ -494,6 +495,25 @@ int load_env_file_pairs(FILE *f, const char *fname, char ***ret) {
         return 0;
 }
 
+int load_env_file_pairs_fd(int fd, const char *fname, char ***ret) {
```

Comment:
there's one case where load_env_pairs() is currently used in extract_image_and_extensions() which should be ported over to this as well